clevertap-core/src/main/java/com/clevertap/android/sdk/DeviceInfo.java

```diff
@@ -186,7 +187,13 @@ private String getNetworkType() {
         }
 
         private boolean getNotificationEnabledForUser() {
-            return NotificationManagerCompat.from(context).areNotificationsEnabled();
+            boolean isNotificationEnabled;
```

Comment:
modify boolean value here only if we get a genuine false. Catch exception and print logs